packages/core/src/index.html

```diff
@@ -1,6 +1,25 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <script>
+        try {
+            performance.mark('started');
+            const observer = new PerformanceObserver(function(list) {
```

Comment:
This may not work for all browsers considering it's not transpiled by Babel.

Reply:
@aaron-binary yeah you're right, what will you suggest?

Reply:
You can replace `const` with `var` 😆

Reply:
@aaron-binary No that will not fix our problem, because PerformanceObserver itself is not supported in many browsers :D That's why we tried to handle errors with catch. And it will provide  FP metric that is not really important for us. So if a browser doesn't support const is definitely not supporting PerformanceObserver too :laughing: really there is no difference man :laughing:

Reply:
Haha you're absolutely right. 👍